tutorials/install-parse-server/index.mdx

```diff
@@ -312,4 +311,4 @@ Parse server comes with a Dashboard for managing your Parse server applications.
 1. Access the Parse server dashboard by visiting the URL `http://your-server-ip:4040`
     <Lightbox src="scaleway-parse_dashboardlogin.webp" alt="" />
 2. Add the credentials that you entered in the `parse-darshboard-config.json` file.
-    <Lightbox src="scaleway-parse_dashboard.webp" alt="" />
+    <Lightbox src="scaleway-parse_dashboard.webp" alt="" />
```

Comment:
Are you sure about the image name?

Reply:
Idk where this came from. But the image exists and is currently featured in the tutorial, so I guess I'll leave it here?